pcsx2/gui/Panels/CpuPanel.cpp

```diff
@@ -109,7 +109,7 @@ Panels::AdvancedOptionsVU::AdvancedOptionsVU( wxWindow* parent )
 Panels::CpuPanelEE::CpuPanelEE( wxWindow* parent )
 	: BaseApplicableConfigPanel_SpecificConfig( parent )
 {
-	*this	+= Text( pxE( L"Notice: Most games are fine with the default options. ")
+	*this	+= Text( pxE( L"Notice: Most games are fine with the default options.")
```

Comment:
Is it really an improvement ? Is it worth it ? I mean all people will need to revalidate the translation.